src/components/panel/PanelContent.js

```diff
@@ -1,8 +1,28 @@
 import React from 'react';
+import { object } from 'prop-types';
+import classNames from 'classnames/bind';
 import styles from './assets/panelcontent.scss';
 
-const Content = ({ children }) => {
-  return <section className={styles.mainContent}>{children}</section>;
+const cx = classNames.bind(styles);
+
+const Content = ({ children, classes }) => {
+  const contentClasses = cx({
+    mainContent: true,
+    [classes.content]: classes.content,
```

Comment:
Could you rename this to `classes.root`? Just to be consistent with the rest of the components.

Reply:
Right, but I was thinking to always have a `root` class for the root element on every component, then inner elements can have any other name.

The backdrop of the panel is in a different component, right?